scrapbook-gateway-service/routes/album.py

```diff
@@ -2,12 +2,13 @@
 from flask import Blueprint, request, jsonify
 from service_utils import session_service 
 from config import IMAGE_SERVICE_URL__DEV
+import json
 import sys
 
-album_user_api = Blueprint('album_user_api', __name__)
+album_create_api = Blueprint('album_create_api', __name__)
```

Comment:
change this to `album_api`. There should be only 1 Blueprint per file that points to the main Flask app.